src/components/auth/Login.js

```diff
@@ -41,8 +53,15 @@ function Login() {
       {user ? (
         <CircularProgress size="24px" color="inherit" />
       ) : (
-        <Typography component="h1" variant="subtitle1">
-          Google
+        <Typography component="h4" variant="subtitle1">
+          <div className={classes.flex}>
+            <img
+              alt="google icon"
+              src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg"
+              className={classes.googleimg}
+            />
+            <p> Google</p>
+          </div>
         </Typography>
```

Comment:
You can replace `<p>` with `<Typography>` and remove the outer Typography tag otherwise it will show a warning in console: `<div> cannot appear as a descendant of <p>`
```suggestion
          <div className={classes.flex}>
            <img
              alt="google icon"
              src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg"
              className={classes.googleimg}
            />
            <Typography component="h4" variant="subtitle1"> Google</Typography >
          </div>
```